test/goldens/end2end/_dummy_pkg-1.0.0-null-safety.1.json_report.md

```diff
@@ -78,6 +78,10 @@ Could not determine supported platforms as package resolution failed.
 Run `dart pub get` for more information.
 </details>
 
+### [x] 0/0 points: WASM readiness
+
+Unable to detect compatibility with runtime `wasm`.
```

Comment:
We need a reason here. Is it because the resolution failed?

Reply:
Either the resolution failed or there are error-level issues in the code, which prevent static analysis. As these are exposed at the other parts of the report, I don't think we need to repeat them here.